Dockerfile

```diff
@@ -1,19 +1,15 @@
-FROM registry.centos.org/centos/centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 
 LABEL maintainer="Dipanjan Sarkar <dsarkar@redhat.com>"
 
 ENV APP_DIR='/fabric8-analytics-bigquery-manifests-job'
-
-RUN yum install -y epel-release &&\
-    yum install -y gcc git python36-pip python36-devel &&\
-    yum clean all &&\
-    mkdir -p ${APP_DIR}
+RUN microdnf install python3 git && microdnf clean all && mkdir -p ${APP_DIR}
 
 WORKDIR ${APP_DIR}
 
-RUN pip3 install --upgrade pip
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-rudra.git@4cfac114208cd0f062a1d7ab0e03bb6bc7a490a3#egg=rudra
+# Need to use 19.2.3 version only as some of packages are deprecated in latest PIP version.
+RUN pip3 install pip==19.2.3
```

Comment:
--no-cache-dir is must to have pip option to reduce layer size during container image build.
https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/Dockerfile#L16